test/auth/oidc_test.go

```diff
@@ -41,7 +43,7 @@ import (
 	"knative.dev/eventing/test/rekt/resources/sequence"
 )
 
-func TestBrokerSupportsOIDC(t *testing.T) {
+func TestBrokerSupportsOIDCUnderTLS(t *testing.T) {
```

Comment:
Note: Change the test name to include the keyword TLS so that prow will enable TLS for this.